arm-resources/resources/2016-09-01/swagger/resources.json

```diff
@@ -1267,7 +1267,155 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GenericResource"
+            }
+          }
+        }
+      }
+    },
+    "/{resourceId}": {
```

Comment:
@amarzavery I don't think this belongs under ARM. Not all RPs implement this API, and this is not in our RPC contract as well. ARM acts as a proxy and just forwards the /usages request to the respective RP. We wouldn't know what the response definition is, since it is upto the RP to define the response.